src/controller/expenses.controller.js

```diff
@@ -0,0 +1,107 @@
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const { userId, from, to, categories } = req.query;
+
+  if (!userId && !categories && (!from || !to)) {
+    res.send(expensesService.getAllExpenses());
+  }
```

Comment:
This condition seems to be incorrect. It checks if `userId`, `categories`, `from` and `to` are not provided, but it should check if any of them is provided. Consider changing the `&&` operators to `||`.